src/style-transform.mjs

```diff
@@ -14,11 +14,18 @@ export default function styleTransform(options) {
   }
 }
 
+// Certain rule blocks require `changeRules` to run recursively on their contents in order to
+// emit rulesets with custom element names prepended to the selector
+const recursiveTypes = [
+  'container',
+  'media',
+  'supports',
+]
+
 function processBlock({
   css = '',
   scopeTo = '',
   disabled = false,
-  instance = ''
```

Comment:
@ryanbethel pretty sure you added this for instance specific styling. Is it still needed? Do we have an exam of where this is used?

Reply:
Yes, I did add it for instance-based styles in my prototype. But then we changed the implementation and it didn't use it. Now instance based style transforms are handled outside of the scope of that so it can be removed.